tests/smoke/Commands/repl.smoke.yaml

```diff
@@ -1,6 +1,26 @@
 working-directory: ./../../../tests/
 
 tests:
+  - name: repl-def
+    command:
+      - juvix
+      - repl
+      - ../examples/milestone/HelloWorld/HelloWorld.juvix
+    stdin: ":def Nat"
+    stdout:
+      contains: "zero"
+    exit-status: 0
```

Comment:
If I run this one, I get:

```
HelloWorld> :def Nat
builtin nat type Nat =
  zero
  | suc Nat
```
I'd expect the actual def. in concrete syntax. 

```
HelloWorld> :def Nat
builtin nat
type Nat :=
  | zero : Nat
  | suc : Nat → Nat;
```

Reply:
In this case, perhaps, it's better to suggest using parenthesis. 

```
HelloWorld> :def +
/repl:1:1-2: error:
Error solving infixities
  +
```

Reply:
> In this case, perhaps, it's better to suggest using parenthesis.
> 
> ```
> HelloWorld> :def +
> /repl:1:1-2: error:
> Error solving infixities
>   +
> ```

Currently, I am parsing expressions, but I should probably parse identifiers. That way, the parentheses won't be needed

Reply:
both problems have now been addressed